### ISSUE TITLE
controller loadDocument parameter not used

### ISSUE THREAD

**Issue by boskokg** [rbcprolabs/packages.flutter#55]
Can you please explain me how the parameter is used in the loadDocument method:

`
  /// Load document
  Future<void> loadDocument(Future<PdfDocument> documentFuture) {
    if (_pdfViewState != null) {
      return null;
    }
    _pdfViewState._changeLoadingState(_PdfViewLoadingState.loading);
    return _loadDocument(document);
  }

`

**Comment by SergeShkurko** [comment]
https://github.com/rbcprolabs/packages.flutter/blob/master/packages/native_pdf_view/README.md#open-another-document

**Comment by SergeShkurko** [comment]
```dart
pdfController.openDocument(PdfDocument.openAsset('assets/sample.pdf'));
```

**Comment by SergeShkurko** [comment]
I saw a mistake, I'll fix it tomorrow

**Comment by SergeShkurko** [comment]
fixed in 3.0.1
